File: code/codes/js/write_post.js
const ACCESS_TOKEN = "발급받은 ACCESS_TOKEN 입력";
const HOST = "https://openapi.band.us";
const API = "/v2.2/band/post/create";

// parameters
const BAND_KEY = "검색할_밴드_키";
const CONTENT = "글 내용";
const DO_PUSH = false;

const xhr = new XMLHttpRequest();

xhr.onload = function () {
  if (xhr.status != 200) {
    // Error!
    return;
  }
  var responseObj = xhr.response;
  console.log(responseObj); // response!
};
xhr.open(
  "POST",
  `${HOST}${API}?access_token=${ACCESS_TOKEN}&band_key=${BAND_KEY}&content=${CONTENT}`
);
xhr.setRequestHeader("Content-Type", "application/json");
xhr.responseType = "json";
xhr.send();

File: code/codes/ruby/write_post.rb
require 'pp'
require_relative '0_env'

# https://developers.band.us/develop/guide/api/write_post

# 글 쓰기
# 특정 밴드에 글을 작성합니다.

# ポスト作成
# 特定のBANDにポストを作成。

params = {
    'band_key' => 'INPUT YOUR BAND KEY',
    'content' => "한글 日本語\nアイウエオ\n\nあいうえお\n\n\nEnglish"
}
response = post_request('https://openapi.band.us/v2.2/band/post/create', params)
pp response

result_data = response['result_data']
properties = ['band_key', 'post_key']
puts validate?(result_data, properties)